app/utils/auth/auth_utils.py

```diff
@@ -49,6 +49,14 @@ def get_token_data(
             status_code=status.HTTP_403_FORBIDDEN,
             detail="Token has expired",
         ) from error
+    except DecodeError as error:
+        hyperion_access_logger.exception(
+            f"Get_token_data: Failed to decode a token ({request_id})",
+        )
+        raise HTTPException(
+            status_code=status.HTTP_403_FORBIDDEN,
+            detail="Could not validate credentials",
+        ) from error
```

Comment:
Why is the from error required here ?

Reply:
It is currently recommended by Python's Code of Conduct when using try except